boards/st/nucleo_l476rg/nucleo_l476rg.dts

```diff
@@ -63,6 +63,7 @@
 	div-m = <1>;
 	mul-n = <20>;
 	div-r = <4>;
+	div-q = <2>;
```

Comment:
`div-q = <8>;`    will give a PLL Q <=48MHz which is suitable for the RNG clock source

Reply:
Thank you for the notice. I fixed it.